### PR TITLE
SKantor solution

### PR DESCRIPTION
**Pull request by KantorSerhiy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,59 @@
...
+            "Android studio",
+        ])
+
+    def create_smooth_mobile_app(self):
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper,):
+    def __init__(self, name: str):
```
Comment (IvanRamyk) [inline app/main.py:53]: if the constructor is absent in class the parent's one is called by default 

so this constructor is extra

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
remove constructor from the last class

File: app/main.py (hunk 1 above)
Comment (KantorSerhiy) [reply app/main.py]: understand, thank you!

**Review by IvanRamyk — APPROVED** [review]
Well done!

**Pull request closed without merging**